clang/lib/CodeGen/CGLoopInfo.cpp

```diff
@@ -692,15 +702,15 @@ LoopInfo::LoopInfo(BasicBlock *Header, const LoopAttributes &Attrs,
       !Attrs.SYCLMaxInterleavingNInvocations &&
       !Attrs.SYCLSpeculatedIterationsNIterations &&
       Attrs.SYCLIntelFPGAVariantCount.empty() && Attrs.UnrollCount == 0 &&
-      !Attrs.SYCLMaxReinvocationDelayNCycles &&
-      Attrs.UnrollAndJamCount == 0 && !Attrs.PipelineDisabled &&
-      Attrs.PipelineInitiationInterval == 0 &&
+      !Attrs.SYCLMaxReinvocationDelayNCycles && Attrs.UnrollAndJamCount == 0 &&
+      !Attrs.PipelineDisabled && Attrs.PipelineInitiationInterval == 0 &&
```

Comment:
Clang-format fails with these unrelated changes. 
This fixes lint errors with the current patch

https://github.com/intel/llvm/actions/runs/4855756170/jobs/8654627505?pr=9263